expression/builtin.go

```diff
@@ -237,6 +240,21 @@ func (b *baseBuiltinFunc) vectorized() bool {
 	return false
 }
 
+func (b *baseBuiltinFunc) vectorizedChildren() bool {
+	flag := b.vectorizedAllFlag.Load()
```

Comment:
I do not understand why don't we put the implementation into `baseBuiltinFunc.vectorized`?

Reply:
Because each built-in function has the different implementation of `vectorized()` (some return `true` while some return `false`).
The implementation of this method `vectorizedChildren()` for all built-in functions are the same, so I split it into two methods.